### PR TITLE
feat(website/home): redesign home and change global font

### PR DESCRIPTION
**Pull request by gael-boyenval**
## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [x] Yes
- [ ] No

## Does this PR introduce a breaking change?

read [What is a breaking change ?](https://mozaic.adeo.cloud/Contributing/Prerequisite/GitConventions/#breaking-changes-)

- [ ] Yes
- [x] No

## Describe the changes

Github issue number or Jira issue URL: https://design-system-adeo.atlassian.net/browse/MZC-337

### CODE REVIEW

File: packages/tokens/build/scss/_tokens.scss [hunk 1]
```diff
@@ -1,3 +1,5 @@
+
+
 $border-s: 1 !default;
```
Comment (mohamedMok) [inline packages/tokens/build/scss/_tokens.scss]: Any reason for those spaces?

File: src/pages/index.js [hunk 2]
```diff
@@ -1,271 +1,416 @@
...
-            to="/GetStarted/"
-            className="button button--primary hero__button"
-            title="Principles, designer and developper guide"
-          >
-            Getting started
-          </Link>
-          {'  '}
-          <DesignerKitLink
+        </Logo>
+        <PageTitle>mozaic</PageTitle>
```
Comment (mohamedMok) [inline src/pages/index.js]: Maybe `Mozaic` rather than `mozaic`

**Comment by gael-boyenval** [comment]
@tiloyi yeah I heard that, you can review the code

File: src/pages/index.js [hunk 3]
```diff
@@ -1,271 +1,433 @@
...
-              >
-                Download the design kit
-              </DesignerKitLink>
-            </div>
-          </Tile>
-        </div>
-      </Section>
-    </Slice>
-  </main>
+            Join us on slack
```
Comment (caqueste) [inline src/pages/index.js]: ~Join us on slack~ Join us on Slack

**Comment by gael-boyenval** [comment]
@nherwegh : yes sure, it's already 16, but I will reduce the line-height.

**Comment by nherwegh** [comment]
> @nherwegh : yes sure, it's already 16, but I will reduce the line-height. It will render the entire design awesome and professional. ;)

image

I think you are lying 😜

**Comment by gael-boyenval** [comment]
> > @nherwegh : yes sure, it's already 16, but I will reduce the line-height. It will render the entire design awesome and professional. ;)
> 
> image
> 
> I think you are lying 😜

lol sure I'm not. 🥰

**Pull request merged**